pkg/controller/knicluster/knicluster_controller.go

```diff
@@ -319,3 +325,16 @@ func newOperatorGroup(namespace string) *olmv1.OperatorGroup {
 		},
 	}
 }
+
+func getKNICustomResourceNames() (string, string) {
+	var name, namespace string
+
+	// get name
+	if name, ok := os.LookupEnv(KNIClusterNameEnv); !ok {
+		name = KNIClusterNameDefault
+	}
+	// get namespace
+	if namespace, ok := os.LookupEnv(KNIClusterNamespaceEnv); !ok {
+		namespace = KNIClusterNamespaceDefault
```

Comment:
Not sure there is a way to discover our namespace. Do we want to fail if we don't **know** the namespace?

Reply:
Yes. I think checking for it to be set in main.go and exiting with an error if not would be reasonable.